osfoffline/application/background.py

```diff
@@ -19,6 +19,7 @@ def __init__(self):
         self.paused = True  # start out paused
         self.running = False
         self.poller = None
+        self.observer = None
```

Comment:
Ah, I see: it was always going to be set later; this just makes sure we include this attribute as part of the definition of the class.

Very helpful- thanks! :octopus: